lib/tasks/redis_tasks.rake

```diff
@@ -0,0 +1,35 @@
+namespace :redis do
+  task :set_keys_expiration => :environment do
+    stripe_status = ENV['STRIPE_STATUS'] || 'live'
+    raise "Invalid STRIPE_STATUS. Only allowed values are: 'test' and 'live'" unless %w(test live).include?(stripe_status)
+
+    # Expire organization aggregation keys
+    Organization.find_each do |organization|
+      expire_old_keys_for_stat(organization.total_raised_key)
+      expire_old_keys_for_stat(organization.total_charges_count_key)
+    end
+
+    # Expire tag aggregation keys
+    Tag.find_each do |tag|
+      expire_old_keys_for_stat(tag.total_charges_count_key(stripe_status))
+      expire_old_keys_for_stat(tag.total_raised_amount_key(stripe_status))
+    end
+
+    # Expire tag namespace aggregation keys
+    TagNamespace.find_each do |tag_namespace|
+      expire_old_keys_for_stat(tag_namespace.total_charges_count_key(stripe_status))
+      expire_old_keys_for_stat(tag_namespace.total_raised_amount_key(stripe_status))
+    end
+  end
+
+  def expire_old_keys_for_stat(stat)
```

Comment:
Should this print some output so we can see the progress when running the rake task?